pkg/policy/k8s/service.go

```diff
@@ -118,7 +103,7 @@ func (p *policyWatcher) resolveToServices(key resource.Key, cnp *types.SlimCNP)
 	// We consult the service cache to obtain the service endpoints
 	// which are selected by the ToServices selectors found in the CNP.
 	p.svcCache.ForEachService(func(svcID k8s.ServiceID, svc *k8s.Service, eps *k8s.Endpoints) bool {
-		if !p.isSelectableService(svc) {
+		if svc == nil {
```

Comment:
can `svc` ever be nil? I hope not!

Reply:
So you're saying I don't need to check if it's nil, right?